### PR TITLE
Add vue-color@2.0.6 w/ npm auto-update

### PR DESCRIPTION
**Pull request by sufuf3**
Pull request for issue: #10503
Related issue(s): # #
@cdnjs/intern3 please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/xiaokaike/vue-color
 * Official website (optional, not the repository): https://xiaokaike.github.io/vue-color/
 * NPM package url (optional): https://www.npmjs.com/package/vue-color
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 8
   - Count of stars: 215
   - Count of forks: 40
 * NPM download stats (optional):
   - Downloads in the last day: 24
   - Downloads in the last week: 187
   - Downloads in the last month: 1096

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sufuf3** [comment]
@hare1039  the `.vue` files can't minify.
but I minify other files.

File: ajax/libs/vue-color/0.0.1/app.min.js [hunk 1]
```diff
@@ -0,0 +1 @@
+// Error: Unexpected token: name (color)
```
Comment (hare1039) [inline ajax/libs/vue-color/0.0.1/app.min.js]: @sufuf3 Something strange happened in here.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by sufuf3** [comment]
@extend1994 @hare1039  I remove v0.0.1
Please help me review this PR, thanks.

**Comment by sufuf3** [comment]
ping @extend1994 @clairetsai818

**Pull request merged**